packages/gp2-components/src/organisms/OutputCard.tsx

```diff
@@ -146,6 +155,11 @@ const OutputCard: React.FC<OutputCardProps> = ({
           />
         )}
       </div>
+      {showTags && tags.length > 0 && (
+        <div css={tagListContainerStyles}>
+          <TagList tags={tags.map((tag) => tag.name)} />
```

Comment:
Also limited to 3 tags
<img width="374" alt="Screenshot 2024-03-25 at 14 18 19" src="https://github.com/yldio/asap-hub/assets/16595804/f8a567e9-a15f-428c-92b8-452c8196fcae">